satpy/tests/compositor_tests/__init__.py

```diff
@@ -777,11 +784,12 @@ def test_call_with_mock(self, match_data_arrays, check_times, combine_metadata,
         match_data_arrays.assert_called_once()
 
     def test_call(self):
-        """Test calling generic compositor"""
+        """Test calling generic compositor."""
         # Multiple datasets with extra attributes
         all_valid = self.all_valid
         all_valid.attrs['sensor'] = 'foo'
-        attrs = {'foo': 'bar'}
+        attrs = {'foo': 'bar', 'resolution': 333}
+        self.comp.attrs['resolution'] = None
```

Comment:
Wouldn't a more accurate test be that resolution is the same across all the inputs (prerequisites)?

Reply:
that would be tested in the test for `combine_metadata` right ? What I'm proposing here is to check that the resolution is preserved even if the composite's attributes have a resolution set to None. (Which was the bug I found)

Reply:
I suppose. I think `resolution` being specified in the `__call__` is less likely than wanting to preserve it from the input data.